templates/hooks/subnetgroup/sdk_read_many_pre_set_output.go.tpl

```diff
@@ -1,8 +1,8 @@
-var subnets []*string
-for _, subnet := range resp.SubnetGroups[0].Subnets {
-	if  subnet.Identifier != nil{
-		subnets = append(subnets, subnet.Identifier)
-	}
-}
+    var subnets []*string
```

Comment:
nit: Is this tab space required?

Reply:
Almost all other controllers have tab in template files, so I guess that's the format?

Reply:
It is not required, but this is the correct indentation